apps/zbugs/shared/queries.ts

```diff
@@ -55,15 +73,23 @@ export const queries = {
     builder.user.where('id', userID).one(),
   ),
 
+  crewUser: defineQuery(idValidator, ({args: userID}) =>
+    builder.user.where('id', userID).where('role', 'crew').one(),
+  ),
+
   labels: defineQuery(
     z.object({
       projectName: z.string(),
     }),
+    labelsOrderByName,
+  ),
 
-    ({args: {projectName}}) =>
-      builder.label.whereExists('project', q =>
-        q.where('lowerCaseName', projectName.toLocaleLowerCase()),
-      ),
+  labelsOrderByName: defineQuery(
+    z.object({
+      projectName: z.string(),
+      orderBy: z.enum(['asc', 'desc']),
```

Comment:
`orderBy` not used. I think we can just change `labels` query to always return ordered by name. We don't need both queries.